tests/unit/server/api/routers/v1/test_prompts.py

```diff
@@ -0,0 +1,63 @@
+from secrets import token_hex
+
+import httpx
+from strawberry.relay import GlobalID
+
+from phoenix.db import models
+from phoenix.server.api.types.node import from_global_id_with_expected_type
+from phoenix.server.api.types.Prompt import Prompt
+from phoenix.server.api.types.PromptVersion import PromptVersion
+from phoenix.server.types import DbSessionFactory
+
+
+class TestPromptsEndpoint:
```

Comment:
```suggestion
class TestPrompts:
```
Nit: I assume there will be more than one endpoint in this test class eventually.